en/asgardeo/docs/complete-guides/javascript/accessing-protected-api.md

```diff
@@ -0,0 +1,99 @@
+---
+template: templates/complete-guide.html
+heading: Accessing protected API from your JavaScript app
+read_time: 2 min
+---
+
+In this section, we will focus on how to call a secure API from your JavaScript app using the other token—the access token.
```

Comment:
`other token—the access token.` shall we make this clearer?

Reply:
Can you plz send a PR?

Reply:
Sure. Will create a PR.